### PR TITLE
Upgrade Go Modules

### PR DESCRIPTION
**Pull request by chimanjain**
# Description

Upgrade go modules.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/1221 |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [x] Ran Sanity test (ran scalability and longevity test on csi-powerscale driver)

### CODE REVIEW

**Review by AkshaySainiDell — APPROVED** [review]
lgtm

File: go.mod [hunk 1]
```diff
@@ -3,22 +3,23 @@ module github.com/dell/cert-csi
 go 1.22
 
 require (
-	github.com/cheggaaa/pb/v3 v3.0.2
-	github.com/dell/csm-replication v1.7.1
-	github.com/fatih/color v1.15.0
+	github.com/cheggaaa/pb/v3 v3.1.5
+	github.com/dell/csi-volumegroup-snapshotter v1.5.1
+	github.com/dell/csm-replication v1.8.0
```
Comment (adarsh-dell) [inline go.mod:8]: Last release of replication:

https://github.com/dell/csm-replication/releases/tag/v1.8.1 

Do we need to point out to this specific version?

Comment (chimanjain) [reply go.mod:8]: Sure let me update it

**Review by adarsh-dell — APPROVED** [review]
Now this LGTM.
Thanks @chimanjain

**Review by AkshaySainiDell — APPROVED** [review]
lgtm

**Pull request merged**